### PR TITLE
Better usage info

### PR DESCRIPTION
**Pull request by icyphox**
I made the usage output look a bit more bearable.

### CODE REVIEW

File: cmd/hydroxide/hydroxide.go [hunk 1]
```diff
@@ -283,13 +283,16 @@ func main() {
...
 		log.Fatal(<-done)
 	default:
-		log.Println("usage: hydroxide serve")
-		log.Println("usage: hydroxide smtp")
-		log.Println("usage: hydroxide imap")
-		log.Println("usage: hydroxide carddav")
-		log.Println("usage: hydroxide auth <username>")
-		log.Println("usage: hydroxide export-secret-keys <username>")
-		log.Println("usage: hydroxide status")
+		log.Println(`
```
Comment (emersion) [inline cmd/hydroxide/hydroxide.go]: Can we move the backtick to the next line, to prevent the empty newline before the usage?

Comment (icyphox) [reply cmd/hydroxide/hydroxide.go]: Sure, but that looks like this:

```
2019/07/27 10:52:09 usage: hydroxide <command>
commands:
	auth <username>		Login to ProtonMail via hydroxide
	export-secret-keys	Export keys
	smtp			Run hydroxide as an SMTP server
	imap			Run hydroxide as an IMAP server
	carddav			Run hydroxide as a CardDAV server
	serve			Run all servers
	status			View hydroxide status
2019/07/27 10:52:09 Unrecognized command
```

Perhaps using the `log` module isn't quite right haha.

Comment (emersion) [reply cmd/hydroxide/hydroxide.go]: Oh, right. `fmt` is probably better suited.

**Comment by icyphox** [comment]
I've removed the initial newline. I think the switch to `fmt` can be dealt with in another PR.

**Pull request merged**